server/src/main/resources/db/mysql/migration/V11__add_spiffe_id_to_clients.sql

```diff
@@ -0,0 +1,4 @@
+# https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#23-maximum-spiffe-id-length
+# maximum SPIFFE ID length is 2048 bytes using (per https://tools.ietf.org/html/rfc3986) US-ASCII
+# chars, which in utf-8 encoding occupy one byte each
+ALTER TABLE clients ADD spiffe_id varchar(2048);
```

Comment:
Something I should have caught in the 1st pass, but didn't. `clients.name` has unique index. Do we expect `clients.spiffe_id` to also be unique? If so, then there might be a [Mysql limitation there](https://dev.mysql.com/doc/refman/5.7/en/innodb-limits.html) -- although given appropriate config is in place, this should be fine.

Reply:
We actually don't expect `spiffe_id` to be unique. If this is used alongside Keysync, for instance, we may give a single SPIFFE ID to each Keysync instance, link that ID to all of the clients that Keysync needs to sync secrets for, and use that to retrieve secrets for multiple clients in one call.